# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/TransportSecurityInfo.h
# Commit: 48ce2b670f32
# Full Hash: 48ce2b670f32691b0537c13c165ec8459d74907c
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-09-25 09:36:42
# Regressor Bug: 1577643
# File Overlap Count: 1
# Description:
#   Bug 1577643 - Implement a security info class for the quic transport. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44073
# ==============================================================================

diff -r 2a39731bf265 -r 48ce2b670f32 security/manager/ssl/TransportSecurityInfo.h
--- a/security/manager/ssl/TransportSecurityInfo.h	Tue Sep 24 22:40:20 2019 +0000
+++ b/security/manager/ssl/TransportSecurityInfo.h	Tue Sep 24 20:56:39 2019 +0000
@@ -113,6 +113,7 @@
 
  protected:
   nsCOMPtr<nsIInterfaceRequestor> mCallbacks;
+  nsCOMPtr<nsIX509CertList> mSucceededCertChain;
 
  private:
   uint32_t mSecurityState;
@@ -124,7 +125,6 @@
   OriginAttributes mOriginAttributes;
 
   nsCOMPtr<nsIX509Cert> mServerCert;
-  nsCOMPtr<nsIX509CertList> mSucceededCertChain;
 
   /* Peer cert chain for failed connections (for error reporting) */
   nsCOMPtr<nsIX509CertList> mFailedCertChain;